Jenkinsfile

```diff
@@ -16,7 +16,13 @@ dockerizedBuildPipeline(
   prepare: {
     githubStatusUpdate('pending')
 
-    sh "docker run --name selenium-chrome -d -p 4444:4444 -v /dev/shm:/dev/shm ${seleniumDockerImage}:${seleniumDockerVersion}"
+    sh """
+      docker run --name selenium-chrome -d \
+        -p 4444:4444 \
+        -v /dev/shm:/dev/shm \
+        -e NODE_MAX_INSTANCES=5 -e NODE_MAX_SESSION=5 -e START_XVFB=false \
```

Comment:
It turns out the tests weren't correctly being run in parallel and the mouse moves were conflicting between tests.  The server needs to be configured for parallel runs and the mouse moves don't happen in true headless mode (which doesn't use XVFB)